### ISSUE TITLE
Latest stix2 version can't be imported (relies on `six`)

### ISSUE THREAD

**Issue by ShayNehmad** [oasis-open/cti-python-stix2#422]
Seems like since stix2 updated to `2.0.1` my builds started failing, see https://travis-ci.com/github/guardicore/monkey/builds/174619100#L671

The error: 

```
==================================== ERRORS ====================================
_ ERROR collecting monkey_island/cc/services/attack/test_mitre_api_interface.py _
ImportError while importing test module '/home/travis/build/guardicore/monkey/monkey/monkey_island/cc/services/attack/test_mitre_api_interface.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
monkey_island/cc/services/attack/test_mitre_api_interface.py:3: in <module>
    from monkey_island.cc.services.attack.mitre_api_interface import MitreApiInterface
monkey_island/cc/services/attack/mitre_api_interface.py:3: in <module>
    from stix2 import FileSystemSource, Filter, CourseOfAction, AttackPattern
../../../../virtualenv/python3.7.1/lib/python3.7/site-packages/stix2/__init__.py:29: in <module>
    from .datastore.filesystem import (
../../../../virtualenv/python3.7.1/lib/python3.7/site-packages/stix2/datastore/filesystem.py:11: in <module>
    from stix2 import v20, v21
../../../../virtualenv/python3.7.1/lib/python3.7/site-packages/stix2/v20/__init__.py:17: in <module>
    from .base import (
../../../../virtualenv/python3.7.1/lib/python3.7/site-packages/stix2/v20/base.py:3: in <module>
    from ..base import (
../../../../virtualenv/python3.7.1/lib/python3.7/site-packages/stix2/base.py:19: in <module>
    from .markings import _MarkingsMixin
../../../../virtualenv/python3.7.1/lib/python3.7/site-packages/stix2/markings/__init__.py:22: in <module>
    from stix2.markings import granular_markings, object_markings
../../../../virtualenv/python3.7.1/lib/python3.7/site-packages/stix2/markings/granular_markings.py:6: in <module>
    from stix2.versioning import new_version
../../../../virtualenv/python3.7.1/lib/python3.7/site-packages/stix2/versioning.py:9: in <module>
    from six.moves.collections_abc import Mapping
E   ModuleNotFoundError: No module named 'six.moves.collections_abc'
```

Seems like the issue is caused because `six` is a dependency or something.

My current workaround is to lock version in `requirements.txt` to 1.4.0

**Comment by emmanvg** [comment]
Related #423

**Comment by clenk** [comment]
Hi @ShayNehmad. We just released v2.0.2 which should fix the issue. Thanks.

**Comment by ShayNehmad** [comment]
Damn, quick turnaround! 

![image](https://user-images.githubusercontent.com/48879847/86791280-09b6c880-c072-11ea-998d-be3653bd3295.png)

Tested and it works, thanks!
